src/distributions/base-installer.ts

```diff
@@ -24,20 +25,29 @@ export abstract class JavaBase {
     ));
     this.architecture = installerOptions.architecture;
     this.packageType = installerOptions.packageType;
+    this.checkLatest = installerOptions.checkLatest ?? false;
```

Comment:
Shouldn't need the `?? false` since `getBooleanInput` defaults to it if there is no value